src/shared/lib/utils/highlightText.tsx

```diff
@@ -0,0 +1,23 @@
+// 하이라이트 함수 추가
+export const highlightText = (
+  text: string | null | undefined,
+  highlight: string,
+) => {
```

Comment:
이거 리액트 엘리먼트를 반환하는 함수인데 그냥 컴포넌트로 바꿔보면 어땠을까 싶기도 합니다!
```suggestion
export const HighlightText = ({
  text,
  highlight,
}: {
  text: string | null | undefined;
  highlight: string;
}) => {
```